nvflare/private/fed/client/client_executor.py

```diff
@@ -227,6 +227,7 @@ def check_status(self, job_id):
                     channel=CellChannel.CLIENT_COMMAND,
                     topic=AdminCommandNames.CHECK_STATUS,
                     request=request,
+                    optional=True,
```

Comment:
How do we decide what commands are optional?

Reply:
Commands are not optional. The messages are marked optional for the sole purpose of suppressing unnecessary log cell error messages (they are instead logged at DEBUG level).

Reply:
got it, thanks for the clarification